Misc/NEWS.d/next/Core_and_Builtins/2025-10-17-23-58-11.gh-issue-140272.lhY8uS.rst

```diff
@@ -0,0 +1 @@
+Fix memory leak in ``_gdbm.gdbm.clear`` by properly freeing memory allocated by ``gdbm_firstkey()``.
```

Comment:
```suggestion
Fix memory leak in the :meth:`clear` method of the :mod:`dbm.gnu` database.
```
`gdbm_firstkey()` and `_gdbm.gdbm` are implementation details not known to common Python user. The NEWS entry can be formulated in understandable terms.

The commit message can be more core developer oriented.